src/App.css

```diff
@@ -76,3 +76,10 @@
   height: 60px;
   margin-right: 10px
 }
+
+.facebookIcon {
+  margin-top: 20px;
+}
+.twitterIcon {
+  margin-top: 5px;
+}
```

Comment:
Superb!

can you please increase the margin?

![margin](https://user-images.githubusercontent.com/39244918/66375003-b2da7080-e9c5-11e9-94b1-ca452208c561.png)

Reply:
add margin-top please cherck